src/middleware/AdaptiveCardSubmitAction.ts

```diff
@@ -0,0 +1,24 @@
+import * as builder from "botbuilder";
+import { DialogMatches } from "../utils/DialogMatches";
+
+// Set the text field on message events to "adaptive card", if request is from an adaptive card
+export class AdaptiveCardSubmitAction implements builder.IMiddlewareMap {
```

Comment:
Let's change the name to AdaptiveCardSubmitActionHandler

Reply:
Done.